src/lib/y2storage/filesystems/btrfs.rb

```diff
@@ -119,7 +149,7 @@ def delete_btrfs_subvolume(devicegraph, path)
         devicegraph.remove_btrfs_subvolume(subvolume)
       end
 
-      # Creates a new btrfs subvolume for the filesystem
+      # Create a new btrfs subvolume for the filesystem
```

Comment:
Any reason to change this? In fact, the whole class (except the methods you added and this one you changed) are documented in third person (i.e. with the "s"), following the Ruby convention.

It's not a big deal if some people prefer to not follow the convention when they document a new method. But I would avoid a war in which each side "fixes" the wording used by the other side on every modification of a documented method. :wink:

Reply:
A quick (not representative) check shows it's not consistent; not even your comments within a class cf. `fake_device_factory.rb`.

Reply:
`fake_device_factory.rb` is one of the classes mainly created and documented by @shundhammer, who also doesn't follow the Ruby convention. As said, I'm fine with some people not following it. I just want to avoid that everyone touches what others did to adjust it to his version.